src/lib/components/collections/collections-list/Folder.svelte

```diff
@@ -276,24 +278,118 @@
       disabled: false,
     },
   ];
+  let requestCount: number = 0;
+  let folderCount: number = 0;
+  let deleteIds: string[] = [];
   $: {
     if (activePath) {
       if (activePath.collectionId === collection.id) {
         visibility = true;
       }
     }
+    if (collection) {
+      deleteIds.length = 0;
```

Comment:
Setting the length explicitly is not recommended.

Reply:
Also, it should be deletedIds. The current var looks like it holds a boolean.

Reply:
resolved